src/ray/protobuf/core_worker.proto

```diff
@@ -94,8 +105,12 @@ message PushTaskRequest {
 message PushTaskReply {
   // The returned objects.
   repeated ReturnObject return_objects = 1;
+  // Set to true if the task was stolen before its execution at the worker.
+  bool task_stolen = 2;
+  // The address of the thief if the task was stolen
+  Address thief_addr = 3;
```

Comment:
Do we need this field?

Reply:
It looks like this field was only used for debugging -- so I just removed it!